include/etcpal/cpp/inet.h

```diff
@@ -0,0 +1,658 @@
+/******************************************************************************
+ * Copyright 2019 ETC Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************
+ * This file is a part of EtcPal. For more information, go to:
+ * https://github.com/ETCLabs/EtcPal
+ ******************************************************************************/
+
+/// \file etcpal/cpp/inet.h
+/// \brief C++ wrapper and utilities for etcpal/inet.h
+
+#ifndef ETCPAL_CPP_INET_H_
+#define ETCPAL_CPP_INET_H_
+
+#include <algorithm>
+#include <array>
```

Comment:
I found the comment considering this down below, but if we don't use std::array anywhere the include should be removed.

Reply:
Maybe we can have include-what-you-use help us identify unused includes in the future too?